tests/instrumentation/transactions_store_tests.py

```diff
@@ -217,46 +218,61 @@ def test_get_transaction_clear():
 def test_tag_transaction():
     requests_store = Tracer(lambda: [], lambda: [], lambda *args: None)
     transaction = requests_store.begin_transaction("test")
-    elasticapm.tag(foo="bar")
-    transaction.tag(baz="bazzinga")
+    elasticapm.label(foo="bar")
```

Comment:
I would go as far as duplicating these tests to make sure that the deprecated `tag` methods still work as expected. And while at it, change the method name :grin:

Reply:
yeah, good point!

Reply:
done in 7a8524d9